CMakeLists.txt

```diff
@@ -2,58 +2,82 @@
 cmake_minimum_required (VERSION 2.8)
 project (ConnectSDK)
 
-set (CONNECT_SOURCES
-    src/client.cpp
-    src/curl-session.cpp
-    src/domain-types.cpp
-    src/util.cpp
-    src/const-strings.cpp
-)
+if (DEFINED IOS_PLATFORM)
 
-set(Boost_USE_STATIC_LIBS ON)
-set(Boost_USE_MULTITHREADED ON)
-set(Boost_USE_STATIC_RUNTIME ON)
+    if (NOT DEFINED ENV{CONNECT_PREBUILT_ROOT})
+        message (FATAL_ERROR "Please, define CONNECT_PREBUILT_ROOT environment variable")
+    endif ()
+
+    set (BOOST_IOS_ROOT $ENV{CONNECT_PREBUILT_ROOT}/boost)
```

Comment:
Per our discussion, we shall move out platform specific scripts outside of root CMakefile.txt into platform subdirs